### DISCUSSION TITLE
Issue with Sandbox payment when open a new window

### DISCUSSION THREAD

**Discussion by Gatisvagulis** [community/community#21363] [Other Feature Feedback, Questions, & Ideas]
<p>Hi. I am testing my e-commerce site payment system via sandbox. In my sandbox business account I have set to redirect after purchase to tank_you.php page. only registered clients can purchase something.  But I have interesting issue. If I open new window, login and make purchase before redirecting to tank_you I lose my client login session. If I stay in the same window and login in the same page again and make purchase everything is working properly. What could make such behavior only in new window? What could be the reason for loosing sessions in new window only after redirecting from sandbox?  </p>

**Comment by avolpe98** [comment, marked as answer]
<p>It depends by how sandbox use a session, if it use the session of the browser or for example it pass variable generated every time a page is loaded.</p>
<p>For example , if it use variable for the session, when you enter in payment page, then you load another page of thankyou. But if first page don’t pass the variable used for the session to the second page, when the secon page controls if the variable passed by first page are the same of variable stores in the database it don’t find the variable used for session and it close your session.</p>
<p>Find something like that in your code.</p>
<p>I hope to be useful. Good luck</p>
